sdk/python/feast/feature_store.py

```diff
@@ -161,14 +185,29 @@ def list_feature_services(self) -> List[FeatureService]:
         return self._registry.list_feature_services(self.project)
 
     @log_exceptions_and_usage
-    def list_feature_views(self) -> List[FeatureView]:
+    def list_feature_views(self, allow_cache: bool = False) -> List[FeatureView]:
         """
         Retrieves the list of feature views from the registry.
 
+        Args:
+            allow_cache: Whether to allow returning entities from a cached registry.
+
         Returns:
             A list of feature views.
         """
-        return self._registry.list_feature_views(self.project)
+        return self._list_feature_views(allow_cache)
+
+    def _list_feature_views(
```

Comment:
is there a reason for having a new method here?

Reply:
This method is here because we want `FeatureStore`, not `Registry`, to handle all entityless logic. We also don't want the public method `list_feature_views` to expose `hide_entityless`, so we decided to add this private method.

Reply:
I think we could expect users to use `FeatureStore.list_entities()` (or `list_featureviews()`) to examine the registry as well. So we want to hide it there as well.